### PR TITLE
Life is made of changes isn't it?

### PR DESCRIPTION
**Pull request by dmcrodrigues**
The members of the squads are a bit outdated due the latest changes.

### CODE REVIEW

**Review by mluisbrown — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -61,23 +61,24 @@ At Babylon, we firmly believe that **transparency** is a core value that should
...
-| Women's Health                | Anil                                 |    1/1       | [#womenshealthteam](https://babylonhealth.slack.com/archives/CFX73S4P5) |
-| US Professional Services      | Greg, Sam, Patrick                   |    3/3       | [#us-engineering](https://babylonhealth.slack.com/archives/CEE6CD37E) |
+| Squad Name                    | Members                                     | Availability | Public slack channel |
+|-------------------------------|---------------------------------------------| ------------ | -------------------- |
+| Booking                       | Witold                                      |    1/1       | |
+| Condition Management          | Catarina, João                              |    2/2       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
+| Consultation                  | Ilya, Adrian, Chitra                        |    3/2       | |
+| Core Experience               | Sergey, Yuri                                |    2/2       | [#core-experience](https://babylonhealth.slack.com/archives/CCSE8JLK0) |
```
Comment (karabatov) [inline README.md]:
```suggestion
| Core Experience               | Sergey, Yuri                                |    2/2       | [#core-exp-squad](https://babylonhealth.slack.com/archives/CCSE8JLK0) |
```

**Review by diegopetrucci — APPROVED** [review]

**Comment by karabatov** [comment]
🤕 
image

**Comment by mluisbrown** [comment]
@karabatov the rich diff is not great. If you view the file you will see it's ok. Some of this also comes from the fact that squads were re-ordered to be in alphabetical order.

**Comment by karabatov** [comment]
@mluisbrown yeah, I checked the source, it's fine :)

File: README.md (hunk 1 above)
Comment (karabatov) [reply README.md]: The channel has a new name but the link is the same.

**Review by horothesun — APPROVED** [review]

**Pull request merged**